### PR TITLE
Deploy contracts

### PR DESCRIPTION
**Pull request by njokuScript**
Deploys the latest updates of the connectors to Base Mainnet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,3 +71,10 @@ export RPC_URL=$DESIRED_NETWORK_RPC_URL
 ```
 sh shell/deploy.sh --network=DESIRED_NETWORK
 ```
+
+### Deployed Addresses
+| Contract | Network | Address |
```
Comment (renanmav) [inline README.md:76]: Nice docs!

I'd just add an empty space between heading and table.

```suggestion
### Deployed Addresses

| Contract | Network | Address |
```

File: broadcast/deploy.s.sol/8453/run-1728641679.json
Comment (renanmav) [inline broadcast/deploy.s.sol/8453/run-1728641679.json:1]: @njokuScript is it good practice to commit this `broadcast` folder?

Comment (njokuScript) [reply broadcast/deploy.s.sol/8453/run-1728641679.json:1]: ignoring now.

Comment (njokuScript) [reply broadcast/deploy.s.sol/8453/run-1728641679.json:1]: it's the transaction result from the deployment process.

**Review by renanmav — APPROVED** [review]

**Pull request merged**